submissions/onyemeMichael/introductionToHtml/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
```

Comment:
Well done! You only have to use  2 product images. Delete the images you are not using.